src/mixins/itext_key_behavior.mixin.js

```diff
@@ -98,6 +99,10 @@ fabric.util.object.extend(fabric.IText.prototype, /** @lends fabric.IText.protot
     this.hiddenTextarea && this.hiddenTextarea.focus();
   },
 
+  blur: function () {
+    this.abortCursorAnimation();
```

Comment:
i have the last minute thought and this should be optional.
this.stopCursorOnBlur && this.abortCursorAnimation();
This is something that seems logical but only depending on what your app is doing when you are focusing out.
If i need to add, as an example, an emoji from an emoji panel, i probably want to see my cursor.

Reply:
What do you think?

Reply:
I agree, but the dev can override easily

Reply:
I can add a bit about it